server/src/main/resources/db/changelog/20201103112757-purge-current-sca-certs.xml

```diff
@@ -7,21 +7,38 @@
         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
     <changeSet id="20201103112757-1" author="crog">
+        <validCheckSum>7:b060e85fd3270acbb9778b17c6d0a447</validCheckSum>
         <comment>
             Removes any existing SCA certs to address a problem with cert generation in some organizations
         </comment>
 
         <sql>
             UPDATE cp_cert_serial cs SET revoked = true
-                WHERE EXISTS (SELECT cac.serial_id FROM cp_cont_access_cert cac WHERE cac.serial_id = cs.id);
+            WHERE EXISTS (SELECT cac.serial_id FROM cp_cont_access_cert cac WHERE cac.serial_id = cs.id);
         </sql>
 
         <update tableName="cp_consumer">
             <column name="cont_acc_cert_id"/> <!-- this sets the column value to null -->
         </update>
 
-        <delete tableName="cp_cont_access_cert" />
+        <!--Below changes are made to fix the bug #1980681. -->
+        <dropForeignKeyConstraint baseTableName="cp_consumer" constraintName="fk_cont_acc_cert" />
 
-        <delete tableName="cp_owner_env_content_access" />
+        <sql>
+            TRUNCATE TABLE cp_cont_access_cert;
+            TRUNCATE TABLE cp_owner_env_content_access;
+        </sql>
+
+        <addForeignKeyConstraint
+                baseTableName="cp_consumer"
+                baseColumnNames="cont_acc_cert_id"
+                constraintName="fk_cont_acc_cert"
+                deferrable="false"
+                initiallyDeferred="false"
+                onDelete="NO ACTION"
+                onUpdate="NO ACTION"
+                referencedColumnNames="id"
+                referencedTableName="cp_cont_access_cert"
+                referencesUniqueColumn="false" />
```

Comment:
Something I should have caught on the PR for master: these lines are indented an extra tab width.

Reply:
Done.